src/components/SettingsAdvanced.vue

```diff
@@ -0,0 +1,230 @@
+<template>
+    <div class="kiwi-settings-advanced">
+        <div class="kiwi-settings-advanced-notice">{{ $t('settings_advanced_header') }}</div>
+        <form class="u-form">
+            <div class="kiwi-settings-advanced-filter">
+                <div class="kiwi-settings-filter-container">
+                    <input id="filter" v-model="filterString"
```

Comment:
I can't see a reason why this input has an ID set. Also we don't use IDs on elements anywhere just on the off-chance they conflict with another element of the same ID at some point in future.